contracts/mock/MockValidators.sol

```diff
@@ -0,0 +1,144 @@
+
+    uint256 private constant FIXED1_UINT = 1000000000000000000000000;
+
+    mapping(address => bool) public isValidator;
+    mapping(address => bool) public isValidatorGroup;
+    mapping(address => uint256) private numGroupMembers;
+    mapping(address => uint256) private lockedGoldRequirements;
+    mapping(address => bool) private doesNotMeetAccountLockedGoldRequirements;
+    mapping(address => address[]) private members;
+    mapping(address => address) private affiliations;
+    uint256 private numRegisteredValidators;
+
+    function updateEcdsaPublicKey(
+        address,
+        address,
+        bytes calldata
+    ) external returns (bool) {
+        // silence complier error that this function can be pure
+
+        return true;
+    }
+
+    function updatePublicKeys(
+        address,
+        address,
+        bytes calldata,
+        bytes calldata,
+        bytes calldata
+    ) external returns (bool) {
+        // silence complier error that this function can be pure
+
+        return true;
+    }
+
+    function setValidator(address account) external {
+        isValidator[account] = true;
+    }
+
+    function setValidatorGroup(address group) external {
+        isValidatorGroup[group] = true;
+    }
+
+    function deregisterValidator(uint256 index) external returns (bool) {
+        // solhint-disable-previous-line no-empty-blocks
+    }
+
+    function affiliate(address group) external returns (bool) {
+        affiliations[msg.sender] = group;
+        return true;
+    }
+
+    function deregisterValidatorGroup(uint256 index) external returns (bool) {
+        // solhint-disable-previous-line no-empty-blocks
+    }
+
+    function setDoesNotMeetAccountLockedGoldRequirements(address account) external {
+        doesNotMeetAccountLockedGoldRequirements[account] = true;
+    }
+
+    function meetsAccountLockedGoldRequirements(address account) external view returns (bool) {
+        return !doesNotMeetAccountLockedGoldRequirements[account];
+    }
+
+    function getGroupNumMembers(address group) public view returns (uint256) {
+        return members[group].length;
+    }
+
+    function setNumRegisteredValidators(uint256 value) external {
+        numRegisteredValidators = value;
+    }
+
+    function getNumRegisteredValidators() external view returns (uint256) {
+        return numRegisteredValidators;
+    }
+
+    function setMembers(address group, address[] calldata _members) external {
+        members[group] = _members;
+    }
+
+    function setAccountLockedGoldRequirement(address account, uint256 value) external {
+        lockedGoldRequirements[account] = value;
+    }
+
+    function getAccountLockedGoldRequirement(address account) external view returns (uint256) {
+        return lockedGoldRequirements[account];
+    }
+
+    function calculateGroupEpochScore(uint256[] calldata uptimes) external view returns (uint256) {
+        // silence complier error that this function can be pure
```

Comment:
Is this TODO ?

Reply:
It was an attempt to silence the compiler warning. I emptied the block and suppressed the solhint warnings.